checkov/common/checks_infra/solvers/attribute_solvers/less_than_attribute_solver.py

```diff
@@ -14,4 +15,10 @@ def _get_operation(self, vertex: Dict[str, Any], attribute: Optional[str]) -> bo
         if vertex.get(attribute) is None:  # type:ignore[arg-type]  # due to attribute can be None
             return False
 
+        attr_val = vertex.get(attribute)  # type:ignore[arg-type]  # due to attribute can be None
+        # if this value contains an underendered variable, then we cannot evaluate the check,
+        # so return True (since we cannot return UNKNOWN)
+        if BaseAttributeSolver._is_variable_dependant(attr_val, vertex['source_']):
```

Comment:
You can do this and avoid the import
```suggestion
        if self._is_variable_dependant(attr_val, vertex['source_']):
```
(relevant for all similar changes)